addons/hr_expense/models/hr_expense.py

```diff
@@ -642,18 +667,31 @@ def _get_empty_list_mail_alias(self):
             # encode, but force %20 encoding for space instead of a + (URL / mailto difference)
             params = werkzeug.urls.url_encode({'subject': _("Lunch with customer $12.32")}).replace('+', '%20')
             return Markup(
-                """<p>%(send_string)s <a href="mailto:%(alias_email)s?%(params)s">%(alias_email)s</a></p>"""
+                """<div class="text-muted mt-4">%(send_string)s <a class="text-body" href="mailto:%(alias_email)s?%(params)s">%(alias_email)s</a></div>"""
             ) % {
                 'alias_email': expense_alias.display_name,
                 'params': params,
-                'send_string': _("Or send your receipts at"),
+                'send_string': _("Tip: try sending receipts by email"),
             }
         return ""
 
+    @api.model
+    def get_expense_attachments(self, expense_id):
+        expense = self.env['hr.expense'].search([('id', '=', expense_id)])
+        attachments = [attachment.image_src for attachment in expense.attachment_ids]
+        return attachments
+
     # ----------------------------------------
     # Actions
     # ----------------------------------------
 
+    def action_show_same_receipt_expense_ids(self):
+        self.ensure_one()
+        return self.same_receipt_expense_ids._get_records_action(
+            name=_("Expenses with a similar receipt to %s") % self.name,
```

Comment:
```suggestion
            name=_("Expenses with a similar receipt to %(other_expense_name)s", other_expense_name=self.name),
```
Translation team will thank you